packages/components/card.css

```diff
@@ -0,0 +1,180 @@
+.card {
+    margin: 0.625rem;
+    position: relative;
+    box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 3rem 0px;
```

Comment:
using color variable will be good

Reply:
if its within 0.5rem we can keep that in px , i think 0.625rem is not problem

Reply:
resolved